packages/test/local-server-stress-tests/src/localServerStressHarness.ts

```diff
@@ -777,30 +790,38 @@ async function synchronizeClients(connectedClients: Client[]) {
 	}
 	try {
```

Comment:
Great, these are the ones I wanted covered at least

Reply:
yeah. i went with a light touch on these in areas most likely to hang, as the timeout promises do get in the way occasionally.